hometask3.py

```diff
@@ -0,0 +1,27 @@
+x = input("Enter a symbol:")
+
+list= list(x)
+
+print(list)
+
+for item in list:
+    if item.isdigit():
+        if int(item) % 2 == 0:
+            print(f"{item} is even")
+        else:
+            print(f"{item} is odd")
+    elif item.isupper():
+        print(f"{item} is a capital letter")
+    else:
```

Comment:
You can add here another check:
```
elif item.islower() # for all lowercase letter
    ...
else: # for everything else (symbol)
    ...

Reply:
I've done the changes. Please take them into consideration